docs/cluster_up_down.md

```diff
@@ -112,7 +113,13 @@ $ oc cluster down
 
 ### MacOS with Docker for Mac
 
-1. Install [Docker for Mac](https://docs.docker.com/docker-for-mac/) making sure you meet the [prerequisites](https://docs.docker.com/docker-for-mac/#/what-to-know-before-you-install).
+OpenShift is currently compatible with version 17.09.1 of Docker for Mac.
+
+> `oc cluster up` with Docker for Mac currently does not work due to [issue #17570](https://github.com/openshift/origin/issues/17570).
```

Comment:
This was only the case for the initial 3.7 release of the oc client. The fix should be in 3.7.2 or 3.9.x